tools/functional-tester/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
```

Comment:
it is separated by purpose, to be able to rebuild once or rerun multiple times (static build takes significant time)